apertium-tur-uzb.tur-uzb.lrx

```diff
@@ -1,15 +1,94 @@
 <rules>
+
+  <rule weight ="1">  <!-- default -->
+  <match lemma="ardından" tags="post">
+    <select lemma="ortidan" tags="post"/>
+  </match>
+  </rule>  
+
+  <rule weight ="1">  <!-- default -->
+  <match lemma="çalış" tags="v.tv.*">
+    <select lemma="ishla" tags="v.tv.*"/>
+  </match>
+  </rule>  
+  <rule weight ="2">  <!-- default -->
+  <match lemma="*" tags="v.*.dat" /> <!-- if konuş comes after dative gerund (Ex: tercüme etmeye çalıştı) -->
+  <match lemma="çalış" tags="v.tv.*">
+    <select lemma="harakat qil" tags="v.tv.*"/>
+  </match>
+  </rule>  
+
+
+  <rule weight ="1">  <!-- default -->
+  <match lemma="konuş" tags="v.tv">
+    <select lemma="gapir" tags="v.tv"/>
+  </match>
+  </rule>  
+  <rule weight ="2">
+  <or>
+	<match lemma="*" tags="*.ins" /> <!-- if konuş comes after noun/pronoun+Instrumental(comitative) (Ex: Köpekle konuştum)  -->
+	<match lemma="ile" tags="post" /> <!-- if konuş comes after ile (Ex: Ay ile konuştum)  -->
+  </or>
+  <match lemma="konuş" tags="v.tv.*">
+    <select lemma="gaplash" tags="v.tv.*"/>
+  </match>
+  </rule>  
+
+
+  <rule weight ="1">  <!-- default -->
+  <match lemma="ile" tags="post">
+    <select lemma="bilan" tags="post"/>
+  </match>
+  </rule>  
+
+  <rule weight ="1">  <!-- default -->
+  <match lemma="alt" tags="n">
+    <select lemma="tag" tags="n"/>
+  </match>
+  </rule>  
+
+  <rule weight ="1">  <!-- default -->
+  <match lemma="at" tags="v.tv">
+    <select lemma="ot" tags="v.tv"/>
+  </match>
+  </rule>  
+
+  <rule weight ="1">  <!-- default -->
+  <match lemma="çok" tags="adj">
+    <select lemma="juda" tags="adj"/>
+  </match>
+  </rule>  
+
+  <rule weight="1"> <!-- default -->
+  <match lemma="büyük" tags="adj">
+    <select lemma="katta" tags="adj"/>
```

Comment:
What about adj with other tags, like `cmp` or `subst` (etc.)?

Reply:
So, how to give it within the same rule? "adj.*" does not include adj alone, sparate rules?, nah

Reply:
All right, I learned it now (thanks to spectie). For this case, Imma just drop tags attribute, later I can use <or>